src/EnergyPlus/OutputProcessor.hh

```diff
@@ -74,6 +74,51 @@ struct EnergyPlusData;
 
 namespace OutputProcessor {
 
+    enum class eTimeStepType : int
```

Comment:
Used `eTimeStepType` as a temporary placeholder for a unique name so that I could easily replace it later.  Might rename to just TimeStepType.  Or something else if others are strongly opinionated.